docs/UserGuide.md

```diff
@@ -233,12 +235,21 @@ You can then access the food item later on for editing, deleting etc.
 * `add d/Canned tuna e/1-1-2021 q/1.5 can p/low` adds an item having description `Canned tuna`, expiry date `01-01-2021`, quantity `1.50 can` and priority `LOW`.
 * `add d/Banana pie e/20-11-2020 q/2 p/medium t/$15 t/contains nuts` adds an item having description `Banana pie`, expiry date `20-11-2020`, quantity `2.00 unit`, priority `MEDIUM`, and tags `$15` and `contains nuts`.
 
-The result of executing `add d/Banana pie e/20-11-2020 q/2 p/medium t/$15 t/contains nuts` is shown in Figure 5 and Figure 6.
+Steps:
+1. Type `add d/Banana pie e/20-11-2020 q/2 p/medium t/$15 t/contains nuts` in the _Command Box_.
+2. Press `Enter` to execute.
+
+Before the `add` command is executed:
 
 ![Before executing the add command](images/AddCommandBefore.png)
 
 <p style="text-align: center; text-decoration: underline">Figure 5: The red box shows the add command being typed into the command box.</p>
 
+<br><br>
+Outcome:
```

Comment:
Is there a need for "Outcome:"? Because the next line seems to be more descriptive. Same for the others.

Reply:
To be discussed in meeting